node/user.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2020 - for information on the respective copyright owner
+// see the NOTICE file and/or the repository at
+// https://github.com/direct-state-transfer/dst-go
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package node
+
+import (
+	"github.com/pkg/errors"
+	"perun.network/go-perun/wallet"
+
+	"github.com/direct-state-transfer/dst-go"
+)
+
+// NewUnlockedUser initializes a user and unlocks all the accounts.
+// (those corresponding to on-chain address, off-chain address and all participant addresses.
+func NewUnlockedUser(wb dst.WalletBackend, cfg UserConfig) (dst.User, error) {
+	if wb == nil {
```

Comment:
Also check `cfg` if you need input validation.

Reply:
Input validation is not needed for `cfg`, as it will not cause the program to panic. I have added a test to ensure this.